torchtune/models/phi3/_tokenizer.py

```diff
@@ -101,13 +101,11 @@ def encode(
             trim_leading_whitespace=trim_leading_whitespace,
         )
 
-    def decode(self, ids: List[int], skip_special_tokens: bool = True) -> str:
```

Comment:
Why did you remove `skip_special_tokens`?

Reply:
Weird, maybe accidentally removed. Let me fix

Reply:
Fixed